src/Resources/views/includes/topbar/_actions.html.twig

```diff
@@ -1,5 +1,5 @@
 {% if view is defined and view.actions|length > 0 %}
-    <div class="hidden md:flex lg:ml-3 flex space-x-3 justify-end flex-nowrap items-stretch">
+    <div class="hidden md:flex lg:ms-auto lg:pl-3 flex space-x-3 justify-end flex-nowrap items-stretch">
```

Comment:
Nit: Instead of putting a patting in place that will not work particular well in case the flex wraps, you could add a column-gap or gap property on the flexbox parent that ensures consistent spacing between the individual item containers (e.g. breadcrumbs and actions).

With the left padding, if it wraps, there will still be a left padding even though it might be alone in one row, which is most likely the unwanted responsive behaviour.

Reply:
@renestalder thanks for the hint. I switched to `gap` 👍